docs/use_cases/model_applications/short_range/METdbLoad_fcstFV3_obsGoes_BrightnessTemp.py

```diff
@@ -72,6 +72,19 @@
 # .. highlight:: bash
 # .. literalinclude:: ../../../../parm/use_cases/model_applications/short_range/METdbLoad_fcstFV3_obsGoes_BrightnessTemp.conf
 
+##############################################################################
+# MET Configuration
+# -----------------
+#
+# METplus sets environment variables based on user settings in the METplus
```

Comment:
@lisagoodrich Please replace all of the text in this section with the text "There are no MET tools used in this use case."

Reply:
Done